theano/tensor/signal/downsample.py

```diff
@@ -101,6 +100,9 @@ def out_shape(imgshape, ds, ignore_border=False, st=None):
             extra row/col of partial downsampling (False) or ignore it (True).
         :type ignore_border: bool
 
+        :param padding: pad zeros on four borders of the images
+        :type padding: tuple of two ints 
```

Comment:
Explain that the first int will be the padding both on top and on bottom, and the second int is the padding to the left and to the right.